packages/@n8n/api-types/src/dto/user/role-change-request.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod';
+import { Z } from 'zod-class';
+
+export class RoleChangeRequestDTO extends Z.class({
```

Comment:
Nit - Can we do `-Dto` instead of `-DTO`? Like Nest.js

Reply:
I generally find Pascal case for abbreviations weird. but since we tend to do that a lot anyways, I'll rename these 👍🏽